debian/rules

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --with python2
+
+override_dh_auto_install:
+	install -Dm 644 $(CURDIR)/youtube_dl_gui/data/icons/hicolor/16x16/apps/youtube-dl-gui.png $(CURDIR)/debian/youtube-dl-gui/usr/share/icons/hicolor/16x16/apps/youtube-dl-gui.png
```

Comment:
I am sure you dont need the rules for the PNG icons since the *setup.py* can handle those.